scripts/build-prod.js

```diff
@@ -0,0 +1,134 @@
+const fs = require('fs');
+const fse = require('fs-extra');
+const path = require('path');
+const findRoot = require('./lib/find-root');
+var execSync = require('child_process').execSync;
+
+const paramsVersion = process.argv.find(a => a.startsWith('v'));
+if (!paramsVersion) {
+  throw new Error(`please, specify version as 'vx.x.x'`);
+}
+const versionNumber = paramsVersion.substring(1);
+const zipName = `kmcAnalytics_v${versionNumber}.zip`;
+
+// STEP: check uncommitted changes
+try {
+  execSync('git diff --exit-code');
+}
+catch (error) {
+  console.error('it seems that you have un-commited changes. to perform this action you should either commit your changes or reset them. aborting action');
+  process.exit(1);
+}
+
+const packageRoot = findRoot(process.cwd());
+if (!packageRoot) throw new Error("couldn't find package root");
+
+// STEP: update src/configuration/analytics-config.ts
+const configFileName = "src/configuration/analytics-config.ts";
+const configFilePath = path.resolve(packageRoot, configFileName);
+let configData = fs.readFileSync(configFilePath, 'utf8');
+const regex = /appVersion: '\d\.\d\.\d'/;
+configData = configData.replace(regex, `appVersion: '${versionNumber}'`);
+
+fs.writeFileSync(configFilePath, configData, 'utf-8');
+console.log('analytics-config.ts has been updated!');
+
+// STEP: update package.json
+const packageJsonPath = path.resolve(packageRoot, 'package.json');
+const packageData = fs.readFileSync(packageJsonPath);
+let packageJsonObj = JSON.parse(packageData);
+packageJsonObj.version = versionNumber;
+packageJsonObj = JSON.stringify(packageJsonObj, null, 2);
+
+fs.writeFileSync(packageJsonPath, packageJsonObj);
+console.log('Package.json has been updated!');
+
+// STEP: commit changed files to git
+try {
+  execSync(`git commit -m "bump analytics version to v${versionNumber}"`);
+}
+catch (error) {
+  console.error('it seems that you cannot make a commit');
```

Comment:
Change to "Git commit operation failed. Verify you are logged into Github and have permissions to commit to this repository"